### ISSUE TITLE
code

### ISSUE THREAD

**Issue by lxzkeyan** [xiaoxingzeng/DF2Net#2]
Thank you for sharing the code.But I especially looking forward to the release of the code.

**Comment by xiaoxingzeng** [comment]
> 
> 
> Thank you for sharing the code.But I especially looking forward to the release of the code.

The code and model is released now.

**Comment by lxzkeyan** [comment]
Thanks for sharing the code.But can you share the training code?Finally, I am looking forward to your reply.Thanks.
Email address: lxz_nxu@163.com.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"xiaoxingzeng"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月18日(星期一) 晚上11:28
收件人:&nbsp;"xiaoxingzeng/DF2Net"<DF2Net@noreply.github.com&gt;;
抄送:&nbsp;"云不知深处"<846281856@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [xiaoxingzeng/DF2Net] code (#2)



 
Thank you for sharing the code.But I especially looking forward to the release of the code.
  
The code and model is released now.

**Comment by lxzkeyan** [comment]
The pretraining model and the shape_predictor_68_face_landmarks which &nbsp;website cannot be opened.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"xiaoxingzeng"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月18日(星期一) 晚上11:28
收件人:&nbsp;"xiaoxingzeng/DF2Net"<DF2Net@noreply.github.com&gt;;
抄送:&nbsp;"云不知深处"<846281856@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [xiaoxingzeng/DF2Net] code (#2)



 
Thank you for sharing the code.But I especially looking forward to the release of the code.
  
The code and model is released now.

**Comment by xiaoxingzeng** [comment]
@lxzkeyan  Hi, I add links of baidu drive in the README

**Comment by lxzkeyan** [comment]
Thank you for your reply.Can you share the training code?I'm especially looking forward to your sharing the training code.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"xiaoxingzeng"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月20日(星期三) 上午10:55
收件人:&nbsp;"xiaoxingzeng/DF2Net"<DF2Net@noreply.github.com&gt;;
抄送:&nbsp;"云不知深处"<846281856@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xiaoxingzeng/DF2Net] code (#2)




@lxzkeyan  Hi, I add links of baidu drive in the README

**Comment by liwei46** [comment]
@xiaoxingzeng 

How about the 70k image-depth pair dataset? Thanks.

**Comment by xiaoxingzeng** [comment]
@liwei46 ,  I found that, the size of our training data is about 100G, so maybe I can release the data generation code and upload some examples.

**Comment by xiaoxingzeng** [comment]
@lxzkeyan , It is easy to reimplement our training process, and I upload the most complicate sfs loss related code in [https://github.com/xiaoxingzeng/DF2Net_SfS_Computing](url)
